projects/plugins/jetpack/json-endpoints/jetpack/class.jetpack-json-api-user-create-endpoint.php

```diff
@@ -61,10 +89,17 @@ function create_or_get_user() {
 		return $this->get_user( $user->ID );
 	}
 
+	/**
+	 * Get the user.
+	 *
+	 * @param int $user_id - the user ID.
+	 *
+	 * @return object
```

Comment:
While it's technically redundant, I'd still do like
```suggestion
	 * @return object|WP_Error
```
to make it clear how errors are reported.